packages/client/src/tools.js

```diff
@@ -732,7 +732,7 @@ export default class Tools {
           data.authors.push(author);
           data.authorids.push(authorid);
         }
-      } else if (typeof rawData.author === 'string') {
+      } else if (typeof rawData.author === 'string' || typeof rawData.author === 'object') {
```

Comment:
I would use the following method to get a more reliable type. In case the author is null.
```suggestion
      } else if (this.variableType(rawData.author) === 'string' || this.variableType(rawData.author) === 'object') {
```